lib/moku/task/set_current.rb

```diff
@@ -9,9 +10,9 @@ module Task
     class SetCurrent < Task
 
       def call(release)
-        release.run_per_host(
-          "rm -f #{release.app_path}; " \
-          "ln -s #{target_path(release)} #{release.app_path}"
+        release.run(
+          Sites::Scope.all,
+          "rm -f #{release.app_path}; ln -s #{target_path(release)} #{release.app_path}"
```

Comment:
Shouldn't this be an `&&` rather than `;`? I think the net effect is the same if the removal has some problem, which is also unlikely with `-f`, but the error reported would be about the link, not the removal.

Reply:
This isn't strictly in scope but I think you are correct, and I'll apply this change as well.